
From: Eric W. Biederman <ebiederm@xmission.com>

It is ok to reserve resources > 4G on x86_64 struct resource is 64bit now :)

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/x86_64/kernel/e820.c |    2 --
 1 files changed, 2 deletions(-)

diff -puN arch/x86_64/kernel/e820.c~kexec-e820-64bit arch/x86_64/kernel/e820.c
--- 25/arch/x86_64/kernel/e820.c~kexec-e820-64bit	Fri Aug 20 15:22:06 2004
+++ 25-akpm/arch/x86_64/kernel/e820.c	Fri Aug 20 15:22:06 2004
@@ -185,8 +185,6 @@ void __init e820_reserve_resources(void)
 	int i;
 	for (i = 0; i < e820.nr_map; i++) {
 		struct resource *res;
-		if (e820.map[i].addr + e820.map[i].size > 0x100000000ULL)
-			continue;
 		res = alloc_bootmem_low(sizeof(struct resource));
 		switch (e820.map[i].type) {
 		case E820_RAM:	res->name = "System RAM"; break;
_
